src/benchmarks/micro/libraries/System.Diagnostics/Perf_Activity.cs

```diff
@@ -8,7 +8,6 @@
 
 namespace System.Diagnostics
 {
-    [MemoryDiagnoser]
```

Comment:
why removed the MemoryDiagnoser attribute?

Reply:
@adamsitnik left a comment about that above, I tagged you on it.

Reply:
thanks, I missed this comment.